### PR TITLE
Enable tests for pushes to master and production

### PR DESCRIPTION
**Pull request by jkassemi**
For merge after "Require branches to be up to date before merging" is unchecked.

For performance purposes we'd disabled the requirement to run tests on pushes to
master and production branches, since the PR containing the merge was fully tested
prior to merge. With this option disabled, we re-enable these tests.

### CODE REVIEW

**Comment by aub** [comment]
I think you want to do this for master but not production. We want to continue to require tests to pass before allowing a PR to get merged into production but stop requiring it for master.

**Comment by jkassemi** [comment]
Thanks @aub, updates made. Once this has approval I'll merge and update the repo settings. @crwallace let me know if we need to schedule this or if it's okay to go ahead as soon as it's ready.

**Comment by aub** [comment]
LGTM 👍

**Review by crwallace — APPROVED** [review]

**Pull request merged**
